src/scripts/main.js

```diff
@@ -1,14 +1,33 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  document.querySelector('body')
+    .insertAdjacentHTML('beforeend', `
+      <div
+        class="notification ${type}"
+        style="
+          top: ${posTop}px;
+          right: ${posRight}px;
+        "
+      >
+        <h2 class="title">${title}</h2>
+
+        <p>${description}</p>
+      </div>
+    `);
 };
 
+setTimeout(() => {
+  const notifications = document.querySelectorAll('.notification');
+
+  notifications.forEach(notification => notification.remove());
+}, 2000);
```

Comment:
would be more universal to move this logic inside the pushNotification function and do it for each created notification separately